scripts/apis_scan.py

```diff
@@ -161,8 +163,19 @@ def enable_passive_scanner():
     zap.pscan.enable_all_scanners()
     zap.pscan.disable_scanners(disabledPassiveScan)
 
+def importurls(filepath):
+    """
+    Imports URLs (one per line) from the file with the given file system path.
+    This component is optional and therefore the API will only work if it is installed
+    """
+    return six.next(six.itervalues(zap._request(zap.base + 'exim/action/importUrls',
+                                                         {'filePath': filepath})))
 
 def get_APIs():
+    if urlScan:
+        url_list_path = "/zap" + urlScanDir + "/" + "urlScan.config"
```

Comment:
I don't think that it's a good choice to use concatenation like this.
Consider a case where some of the variables is not a string. In this case you'll get an error:
```bash
>>> "str" + 1
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can only concatenate str (not "int") to str
```

so it's much safer to use a `%` formatter:
```
>>> s = "str"
>>> n = 1
>>> "{}{}".format(s, n)
'str1'
```

or f-strings:
```
>>> s = "str"
>>> n = 1
>>> f"{s} {n}"
'str 1'
```

Do you think it's better to replace the line with f-string formatter?